how-to-docs/drone-how-to.md

```diff
@@ -356,29 +356,39 @@ The build will fail with the following error:
 [ERROR] 2016/10/20 10:17:40 main.go:130: exit status 1
 ```
 
-This suggests that you are not authorised to deploy to the kubernetes cluster. You can fix this by setting your `KUBE_TOKEN` to a valid value. But before you do this, let's recap the environment variables needed to deploy successfully to DSP.
+This suggests that you are not authorised to deploy to the kubernetes cluster. You can fix this by setting your `KUBE_TOKEN` to a valid value. But before you do this, let's recap the environment variables needed to deploy successfully to ACP.
+
+The kube `deploy.sh` script relies on 3 environment variables:
+
+- `KUBE_NAMESPACE` - The kubernetes namespace you wish to deploy to. **You need to provide the kubernetes namespace as part of the deployment job**.
+
+- `KUBE_TOKEN` - This is the token used to authenticate against the kubernetes cluster. **If you do not already have a kube token, you can [submit a support request for a Kubernetes token on the platform hub](https://hub.acp.homeoffice.gov.uk/help/support/requests/new/kubernetes-token)**.
 
-The kube `deploy.sh` scripts relies on 3 environment variables:
+- `KUBE_SERVER` - This is the address of the kubernetes cluster. There are four environment variables set as an organisational secret in Drone: `KUBE_SERVER_DEV`,  `KUBE_SERVER_OPS`, `KUBE_SERVER_PROD` and `KUBE_SERVER_CI`.
 
-- `KUBE_NAMESPACE` - the kubernetes namespace you wish to deploy to. **You need to provide the kubernetes namespace as part of the deployment job**.
 
-- `KUBE_TOKEN` - this is the token used to authenticate against the kubernetes cluster. **You need to add this as a secret to your build step. You can [request a kubernetes token here](https://github.com/UKHomeOffice/application-container-platform-bau/issues/new)**. In this particular case, the secret was added with:
+You will need to add `KUBE_TOKEN` and `KUBE_SERVER` as drone secrets.
 
-  ```bash
-  $ drone secret add --image=quay.io/ukhomeofficedigital/kd:v0.2.3 --conceal  UKHomeOffice/<your_repo> KUBE_TOKEN <your_token>
-  ```
+You can add and view secrets through the Drone UI. Go to Drone and select your repo, then click the icon in the top right and select **Secrets**. You should be presented with a list of the secrets for that repo (if there are any) and you should be able to add secrets giving them a name and value. Add the `KUBE_TOKEN` and `KUBE_SERVER` secrets with their respective values.
```

Comment:
I believe there's already a doc about setting up a Drone build (that @asmith030 updated?) – best to link to that here instead of repeating info.

Reply:
I don't think there is already. If there is, I can't find it.